rotkehlchen/chain/ethereum/modules/lido/decoder.py

```diff
@@ -0,0 +1,126 @@
+            msg_aggregator=msg_aggregator,
+        )
+        self.steth_evm_address = A_STETH.resolve_to_evm_token().evm_address
+
+    def _decode_lido_staking_in_steth(self, context: DecoderContext, sender: ChecksumEvmAddress) -> DecodingOutput:  # noqa: E501
+        """Decode the submit of eth to lido contract for obtaining steth in return"""
+        # decode the ETH collateral amount submitted for staking in Lido
+        amount_raw = hex_or_bytes_to_int(context.tx_log.data[:32])
+        collateral_amount = token_normalized_value_decimals(
+            token_amount=amount_raw,
+            token_decimals=18,
+        )
+
+        # Searching for the already decoded event,
+        # containing the ETH transfer of the submit transaction
+        paired_event = None
+        action_from_event_type = None
+        for event in context.decoded_events:
+            if (
+                event.address == self.steth_evm_address and
+                event.asset == A_ETH and
+                event.balance.amount == collateral_amount and
+                event.event_type == HistoryEventType.SPEND and
+                event.location_label == sender
+            ):
+                event.event_type = HistoryEventType.DEPOSIT
+                event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                event.notes = f'Submit {collateral_amount} ETH to Lido for receiving stETH in exchange'  # noqa: E501
+                event.counterparty = CPT_LIDO
+                #  preparing next action to be processed when erc20 transfer will be decoded by rotki  # noqa: E501
+                #  needed because submit event is emitted prior to the erc20 transfer
+                paired_event = event
+                action_from_event_type = HistoryEventType.RECEIVE
+                action_to_event_subtype = HistoryEventSubType.RECEIVE_WRAPPED
+                action_to_notes = f'Receive {{amount}} stETH in exchange of the deposited ETH'  # {amount} to be set by transform ActionItem  # noqa: E501,F541
+                break
+        else:  # did not find anything
+            log.error(
+                f'At lido steth submit decoding of tx {context.transaction.tx_hash.hex()}'
+                f' did not find the related ETH transfer',
+            )
+            return DEFAULT_DECODING_OUTPUT
+
+        action_items = []  # also create an action item for the reception of the stETH tokens
+        if paired_event is not None and action_from_event_type is not None:
+            action_items.append(ActionItem(
+                action='transform',
+                from_event_type=action_from_event_type,
+                from_event_subtype=HistoryEventSubType.NONE,
+                asset=A_STETH,
+                amount=collateral_amount,
+                amount_error_tolerance=FVal(token_raw_value_decimals(STETH_MAX_ROUND_ERROR_WEI, 18)),  # passing the maximum rounding error for finding the related stETH transfer  # noqa: E501
```

Comment:
Can use the `from_wei` function. Simpler. Also make sure to be rebased on develop so it works with int and returns FVal.

```suggestion
                amount_error_tolerance=from_wei(STETH_MAX_ROUND_ERROR_WEI),  # passing the maximum rounding error for finding the related stETH transfer  # noqa: E501
```